app/vmselect/promql/rollup_result_cache.go

```diff
@@ -263,7 +263,7 @@ func (rrc *rollupResultCache) GetSeries(qt *querytracer.Tracer, ec *EvalConfig,
 	if !ok {
 		mi.RemoveKey(key)
 		metainfoBuf = mi.Marshal(metainfoBuf[:0])
-		bb.B = marshalRollupResultCacheKeyForSeries(bb.B[:0], ec.AuthTokens[0], expr, window, ec.Step, ec.EnforcedTagFilterss)
+		bb.B = marshalRollupResultCacheKeyForSeries(bb.B[:0], at, expr, window, ec.Step, ec.EnforcedTagFilterss)
```

Comment:
It'd be great to have a test for it. 

 Maybe it's a great place to use integration tests suite:

https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/apptest/tests/key_concepts_test.go#L53

@rtm0 WDYT

Reply:
@f41gh7 Need to wait until #7523 is merged. It conains some important updates.

Reply:
Thanks, tested with integration tests. Currently, given panic may occur only hypothetically.  I'll add tests into this PR later.